README.md

```diff
@@ -90,11 +89,15 @@ There are a few different components to this repo:
 - [spirv-std](spirv-std) for GPU intrinsics, types, and other library items used by GPU crates.
 - [spirv-builder](spirv-builder) for a convenient way of building a GPU crate in a CPU build.rs file.
 
-## Getting started
+## Contributing
+
+[![Contributor Covenant](https://img.shields.io/badge/contributor%20covenant-v1.4-ff69b4.svg)](CODE_OF_CONDUCT.md)
+
+We welcome community contributions to this project.
 
-We welcome community contributions to this project. If you would like to get started, be sure to checkout the [`rust-gpu` dev guide][gpu-guide] and our [Contributor Guide](CONTRIBUTING.md) for more information on how to get started.
+If you would like to get started, be sure to checkout the [`rust-gpu` dev guide][gpu-guide] and our [Contributor Guide](CONTRIBUTING.md) for more information on how to get started.
 
-[gpu-guide]: https://embarkstudios.github.io/rust-gpu/book/
+[gpu-guide]: https://embarkstudios.github.io/rust-gpu/
```

Comment:
Shoot, looks like the rebase screwed up here, this wasn't supposed to happen, it should link to the book.

Reply:
I love computers